pkg/fakerp/config.go

```diff
@@ -75,7 +75,7 @@ func GetPluginTemplate() (*pluginapi.Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	rootDockerConfig, err := readFile(filepath.Join(artifactDir, "root-docker-config.json"))
+	redhatIOImagePullSecret, err := readFile(filepath.Join(artifactDir, "redhat-io-docker-config.json"))
```

Comment:
If we renaming this here, lets rename it in https://github.com/openshift/release/pull/2584 too

Reply:
yep, renamed